src/Entity/Cover.php

```diff
@@ -130,6 +137,15 @@ class Cover
     private $updatedAt;
 
     /**
+     * @ApiProperty(
+     *     attributes={
+     *         "openapi_context"={
+     *             "type"="string",
+     *             "example"="870970"
```

Comment:
Note that "870970" is not an agency id for a library in dk?

https://vip.dbc.dk/lister.php?vis=folk&amp;PHPSESSID=7bdb9955c90dca2dbc18228ff54f970b

Reply:
Considered using either `123456` or a real agency id. Decided to use something "real looking" but false. Not sure what happens if we use a real one and people copy paste to much

Reply:
They will nevet be able to set the agency in the request... it will come from the token.

Reply:
This is just an example value. Still don't think we should use a real agency

Reply:
Changed my mind. Now example value is `123456`